File: Mini-Projects/C#/Calculadora.cs
namespace Calculadora
{
    class Math
    {
        static void Main(string[] args)
         

        {
            
            double num1;
            double num2;
            byte operación;
            Console.WriteLine("Digite un número: ");
            num1 = Convert.ToDouble(Console.ReadLine());
            Console.WriteLine("Digite otro número: ");
            num2 = Convert.ToDouble(Console.ReadLine());
            Console.WriteLine("Digite 1 para suma,2 para resta,3 para multiplicación ó 4 para división: ");
            operación = Convert.ToByte(Console.ReadLine());
            switch (operación)
                {
                case 1:
                    Console.WriteLine("El resultado de la suma es : " + (num1 + num2));
                   

                    break;
                case 2:
                   
                    Console.WriteLine("El resultado de la resta es: " + (num1 - num2));
                    break;

                case 3:
                   
                    Console.WriteLine("El resultado de la multiplicación es: " + (num1 * num2));
                    break;
                case 4:
                   
                    Console.WriteLine("El resultado de la división es: " + (num1 / num2));
                    break;
                    default:
                        Console.WriteLine("Elija los números: 1, 2, 3 ó 4 ");
                        break;
                    
                }
            Console.ReadKey();
        }
    }
}

File: Mini-Projects/PYTHON/calculadora.py
import math
#Calculadora
num1 = float(input("Por favor digite un número: "))
num2 = float(input("Por favor digite otro número: "))
#if num1 == -0 or num2 == -0:
    #print("Por favor digite el 0 sin signo")
ope = input("Por favor elija la operación a realizar: suma,resta, multiplicacion o division: ")
print(f"Usted a elegido la siguiente operación: {ope}")
if ope=="suma":
    num = num1 + num2
    print(f"La {ope} es:{num} ")
elif ope =="resta":
    num = num1 - num2
    print(f"La {ope} es:{num} ")
elif ope =="multiplicacion":
    if num1 ==-0 or num2==-0:
       print("La multiplicacion de los 2 nùmeros ingresados da 0")
    else:
        num = num1 * num2
        print(f"La {ope} es: {num}")
if ope == "division":
    if num1 == 0 and num2 == 0:
        print(f"La {ope} de los 2 nùmeros ingresados da error")
    elif num1 > 0 and num2 == 0:
        print(f"La {ope} de los 2 nùmeros ingresados es infinita")
    elif num1 < 0 and num2 == 0:
         print(f"La {ope} de los 2 nùmeros ingresados es -infinita")
    else:
        num = num1 / num2
        print(f"La {ope} es: {num}")